xgi/drawing/xgi_pylab.py

```diff
@@ -205,29 +250,38 @@ def draw_xgi_hyperedges(ax, H, pos, edge_lc, edge_lw, edge_fc, d_max):
             ax.add_patch(obj)
 
 
-def draw_xgi_complexes(ax, SC, pos, edge_lc, edge_lw, edge_fc):
+def draw_xgi_complexes(ax, SC, pos, edge_lc, edge_lw, edge_fc, settings):
```

Comment:
"draw_xgi_simplices" instead?
also push "ax" to 3rd place to match draw()

Reply:
Done.